importer/DkBygningDa.py

```diff
@@ -11,20 +11,22 @@ def set_adm_location(self):
                 self.add_statement("located_adm", adm_location)
 
     def set_location(self):
-        """
-        Use self.municipality because IT'S PLACE NOT MUNICIPALITY
-        (that's adm2)
-        """
         place_item = False
```

Comment:
`place_item = None` is a bit more natural. You can leave the `if place_item` unchanged.